api/test/unit/models/media_report_test.py

```diff
@@ -0,0 +1,138 @@
+import uuid
+from typing import Literal, Union
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from catalog.api.models import (
+    Audio,
+    AudioReport,
+    DeletedAudio,
+    DeletedImage,
+    Image,
+    ImageReport,
+    MatureAudio,
+    MatureImage,
+)
+from catalog.api.models.media import DEINDEXED, DMCA, MATURE, MATURE_FILTERED, PENDING
+
+
+pytestmark = pytest.mark.django_db
+
+MediaType = Union[Literal["audio"], Literal["image"]]
+
+
+@pytest.fixture
+def media_obj():
+    def _get_media_obj(type: MediaType):
+        klass = Audio if type == "audio" else Image
+        identifier = uuid.uuid4()
+        obj = klass(identifier=identifier)
+        obj.save()
+        return obj
+
+    return _get_media_obj
+
+
+@pytest.mark.parametrize(
+    "media_type, report_class", [("image", ImageReport), ("audio", AudioReport)]
+)
+@pytest.mark.parametrize("reason", [DMCA, MATURE])
```

Comment:
This is only used in two tests, but we could potentially set this to a variable for use in multiple tests:

```python
reason_params = pytest.mark.parametrize("reason", [DMCA, MATURE, OTHER])

...

@reason_params
def test_foo()
```

Reply:
Implemented in fafcdf8.